src/java/org/apache/cassandra/hints/HintsDispatcher.java

```diff
@@ -417,7 +417,7 @@ private SplitHint splitHintIntoAccordAndNormal(ClusterMetadata cm, Hint hint)
     {
         SplitMutation<Mutation> splitMutation = ConsensusMigrationMutationHelper.instance().splitMutation(hint.mutation, cm);
         if (splitMutation.trackedMutation != null)
```

Comment:
How does this handle the case where there ends up being no nomral or accord part of the mutation? Won't the callback in the caller never end up firing and it will hang?

Reply:
adjusted to return a null/null split mutation, which will cause the caller to auto complete the callback